Headers/Foundation/NSURL.h

```diff
@@ -97,6 +97,19 @@ GS_EXPORT_CLASS
 #if OS_API_VERSION(MAC_OS_X_VERSION_10_5, GS_API_LATEST)
 + (instancetype) fileURLWithPath: (NSString*)aPath isDirectory: (BOOL)isDir;
 #endif
+
+#if OS_API_VERSION(MAC_OS_X_VERSION_10_11, GS_API_LATEST)
++ (instancetype) fileURLWithPath: (NSString *)aPath
+	                 isDirectory: (BOOL)isDir
+	               relativeToURL: (NSURL *)baseURL;
+#endif
```

Comment:
As these were added for the same release you could use one #if for both methods. The same is true later on for the other two methods.